themes/default/content/blog/announcing-v5.0.0-of-the-pulumi-aws-provider/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Announcing v5.0.0 of the Pulumi AWS Provider"
+date: 2022-03-23
+meta_desc: "v5.0.0 of the Pulumi AWS Provider is now available"
+meta_image: meta.png
+authors:
+- paul-stack
+tags:
+- aws
+---
+
+We are excited to announce v5.0.0 of the Pulumi AWS Provider. Pulumi prides itself on how it continually tries to ensure
+that major version releases are compatible with existing applications. The upstream provider that this is based upon, has had
+a major breaking change in the S3 Bucket resource. In order to ensure that our users are not impacted, We have extended
+our provider to provider both the old implementation and the new implementation of the S3 Bucket resource.
+This means that all existing usages of S3 Bucket will continue to work as normal. We also now offer the ability to
+create a v2 variants of the S3 Bucket.
+
+Let's consider an existing implemention of an S3 Bucket that is used to create a static website:
+
+{{< chooser language "typescript,python,csharp,go" >}}
+
+{{% choosable language typescript %}}
+
+```typescript
+import * as aws from "@pulumi/aws";
+
+const siteBucket = new aws.s3.Bucket("s3-website-bucket", {
+    website: {
+        indexDocument: "index.html",
+    },
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+from pulumi_aws import s3
+
+web_bucket = s3.Bucket('s3-website-bucket',
+                       website=s3.BucketWebsiteArgs(
+                       index_document="index.html",
+                       ))
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using Pulumi;
+using Pulumi.Aws.S3;
+using Pulumi.Aws.S3.Inputs;
+
+class WebsiteStack : Stack
+{
+    public WebsiteStack()
+    {
+        var bucket = new Bucket("my-bucket", new BucketArgs
+        {
+            Website = new BucketWebsiteArgs
+            {
+                IndexDocument = "index.html"
+            }
+        });
+    }
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+package main
+
+import (
+	"github.com/pulumi/pulumi-aws/sdk/v4/go/aws/s3"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		_, err := s3.NewBucket(ctx, "s3-website-bucket", &s3.BucketArgs{
+			Website: s3.BucketWebsiteArgs{
+				IndexDocument: pulumi.String("index.html"),
+			},
+		})
+		if err != nil {
+			return err
+		}
+		
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+{{% /chooser %}}
+
+In the upstream provider, the S3 Bucket resource has been broken into multiple pieces meaning that you would need to move the website
+part of the bucket out to be it's own resource. *We have ensured that this will continue to work in Pulumi.*
```

Comment:
```suggestion
The S3 Bucket resource has been split into multiple pieces in the upstream provider, which means you would need to move the website
part of the bucket out to be its own resource. *We have ensured that this will continue to work in Pulumi.*
```